tools/shell-commands/src/main/resources/requestBody/2.2.0/create_rollover_policy_query.json

```diff
@@ -0,0 +1,19 @@
+{
+  "policy": {
+    "phases": {
+      "hot": {
+        "actions": {
+          "rollover": {
+            #rolloverHotActions
+          }
+        }
+      },
+      "delete": {
+        "min_age": "90d",
+        "actions": {
+          "delete": {}
+        }
+      }
```

Comment:
We really need to not forget about that during the purge story

Reply:
I added a line in Handle TODOs in the https://issues.apache.org/jira/browse/UNOMI-739 ticket to not forget it